internal/restic/node.go

```diff
@@ -790,3 +820,67 @@ func checkGenericAttributeNameNotHandledAndPut(value GenericAttributeType) bool
 	// Key exists, then it is already handled so return false
 	return !exists
 }
+
+// GenericAttributesToOSAttrs gets the os specific attribute from the generic attribute using reflection
+func GenericAttributesToOSAttrs(attrs map[GenericAttributeType]json.RawMessage, attributeType reflect.Type, attributeValuePtr *reflect.Value) (unknownAttribs []GenericAttributeType, err error) {
```

Comment:
It doesn't have to be, but it was causing lint errors like -
func `genericAttributesToOSAttrs` is unused (unused)
func `osAttrsToGenericAttributes` is unused (unused) etc.
because these are helper methods meant to be used by the specific node_xx, they are unused from within node.

I now added // nolint:unused on these functions instead. I didn't want to move them to node_windows because they are meant to be reused for other OSs in future.